charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -33,12 +33,13 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         ports:
-        - containerPort: 6000
+        - containerPort: {{ int .Values.metricsPort }}
           name: metrics
         command:
         - zookeeper-operator
-        {{- if .Values.disableFinalizer }}
         args:
+        - -metrics-bind-address={{ .Values.metricsBindAddress }}:{{ int .Values.metricsPort }}
```

Comment:
Does it have to be `-metrics-bind-address` or `--metrics-bind-address`?

Reply:
Based on: https://github.com/pravega/zookeeper-operator/blob/074d8b09facefb1ec7de7c0ed9142f44cd3d51d8/main.go#L65
And flag's documentation: https://pkg.go.dev/flag#hdr-Command_line_flag_syntax
My understanding is that it can be either. If there is a preference for double dashes, that's easy to adjust, but as `-disableFinalizer` below is a single dash, I went with single dash for consistency.